File: samples/CS/12-FontControl/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using WinForms.Ribbon;
using System.Diagnostics;

namespace _12_FontControl
{
    public partial class Form1 : Form
    {
        private RibbonItems _ribbonItems;
        public RichTextBox RichTextBox1 => richTextBox1;

        public Form1()
        {
            InitializeComponent();
            _ribbonItems = new RibbonItems(_ribbon);
            _ribbonItems.Init(this);
        }

        private void richTextBox1_SelectionChanged(object sender, EventArgs e)
        {
            _ribbonItems.richTextBox1_SelectionChanged(sender, e);
        }
    }
}

File: samples/VB/12-FontControl/Form1.vb
Imports WinForms.Ribbon

Namespace _12_FontControl
    Partial Public Class Form1
        Inherits Form
        Private _ribbonItems As RibbonItems

        Public Sub New()
            InitializeComponent()
            _ribbonItems = New RibbonItems(_ribbon)
            _ribbonItems.Init(Me)

        End Sub
    End Class
End Namespace
